src/components/Tag/Tag.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Chip as MuiChip, ChipProps as MuiChipProps, makeStyles, withTheme } from '@material-ui/core';
+export type TagProps = MuiChipProps;
+
+const useStyles = makeStyles((theme) => ({
+  MuiChip: {
```

Comment:
Styles of the tag doesn't match those from Zeplin - height is too large and typography weight and dimensions are not correct.